csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -621,7 +622,7 @@ Examples:
 
 #### 3.1.3.3 Full Product Name Type - CPE
 
-Common Platform Enumeration representation (`cpe`) of value type `string` with `pattern` (regular expression):
+Common Platform Enumeration representation (`cpe`) of value type `string` of 1 or more characters with `pattern` (regular expression):
```

Comment:
I know that this isn't currently reflected in the schema but shouldn't be the minlength 5?

Reply:
I think that we should be safe by changing it to`minlength 5`